src/pudl/extract/gridpathratk.py

```diff
@@ -0,0 +1,85 @@
+"""Extract GridPath RA Toolkit renewable energy generation profiles from CSVs.
+
+There are several time series available from the GridPath Resource Adequacy Toolkit.
+
+Solar:
+
+* Hourly Solar by Generator
+* Hourly Solar by Zone
+* Hourly Solar by Zone (Extended)
+* Solar Aggregations
+
+Wind:
+
+* Hourly Wind by Plant
+* Hourly Wind by Zone
+* Hourly Wind by Zone (Extended)
+* Wind Aggregations
+
+Thermal Derates:
+
+* Hourly Thermal Derates by Generator
+* Hourly Thermal Derates by Generator (Extended)
+
+Misc:
+
+* Daily Weather Data
+
+"""
+
+from io import BytesIO
+from pathlib import Path
+
+import pandas as pd
+from dagster import AssetsDefinition, Output, asset
+
+# TODO: Reorganize the archive to use asset names for partitions directly
+GRIDPATHRATK_PARTS = {
+    "hourly-solar-zonal-capacity-synth": {
+        "table_name": "aggregated_extended_solar",
+    },
+    "hourly-wind-zonal-capacity-synth": {
+        "table_name": "aggregated_extended_wind",
+    },
+}
```

Comment:
I can change the name of the archive files to match whatever we want to call these so there's no mapping. Hopefully we'll also need to update it to include the thermal plant derates.  I also want to get rid of the top level directory inside the zipfiles. It might also be possible to break the archive down along several real partitions (aggregation / extended or not) but it's not a long series like with the years or states so I kinda don't think it's worth it, and just addressing the individual zipfiles as a unit is fine.

Reply:
I don't have an opinion here, guessing @cmgosnell might. But agree that renaming the files during the download process to match whatever we want here and get rid of this second dictionary seems like a high priority. I could see having partitions that point to level of processing/aggregation (e.g., `generator`, `zonal`, `zonal_extended`), but agreed we can revisit this as we add in more tables.

Reply:
Got rid of this, and renamed the files to reflect the things that are needed to distinguish between the different versions of the data so e.g.:
- `original_solar_capacity.zip`
- `aggregated_solar_capacity.zip`
- `aggregated_extended_solar_capacity.zip`
- `solar_capacity_aggregations.csv`